specs/withdrawal-proof.md

```diff
@@ -0,0 +1,74 @@
+# Withdrawals proof
```

Comment:
Looks like we also need to docs on how to handle balance update on address

Reply:
fixed in [f4e0cfc](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/487/commits/f4e0cfc357419830f179c91cad9aebaf0da8e11a)